tests/FormBuilderTest.php

```diff
@@ -239,13 +239,11 @@ public function testFormTextarea()
         $form2 = $this->formBuilder->textarea('foo', 'foobar');
         $form3 = $this->formBuilder->textarea('foo', null, ['class' => 'span2']);
         $form4 = $this->formBuilder->textarea('foo', null, ['size' => '60x15']);
-        $form5 = $this->formBuilder->textarea('encoded_html', '&amp;');
 
         $this->assertEquals('<textarea name="foo" cols="50" rows="10"></textarea>', $form1);
         $this->assertEquals('<textarea name="foo" cols="50" rows="10">foobar</textarea>', $form2);
         $this->assertEquals('<textarea class="span2" name="foo" cols="50" rows="10"></textarea>', $form3);
         $this->assertEquals('<textarea name="foo" cols="60" rows="15"></textarea>', $form4);
-        $this->assertEquals('<textarea name="encoded_html" cols="50" rows="10">&amp;amp;</textarea>', $form5);
     }
```

Comment:
I think these 2 tests are very important to ensure losslessness. Saving a value shouldn't change it, so saving `&amp;` 4 times, should keep it `&amp;`, not change it into `&`.